.github/workflows/pr-to-master.yaml

```diff
@@ -0,0 +1,22 @@
+name: pr-to-master
+
+on:
+  pull_request_target:
+    types: [opened]
+    branches: [master]
+
+jobs:
+    build:
+        name: Comment on PR submitted to master
```

Comment:
Maybe we could reword to: "Verify a PR was submitted against the correct branch"